c7n/resources/ses.py

```diff
@@ -336,3 +340,189 @@ def process(self, rulesets):
                 RuleSetName=ruleset["Metadata"]['Name'],
                 ignore_err_codes=("CannotDeleteException",)
             )
+
+
+class DescribeSESIngressEndpoint(DescribeSource):
+
+    def augment(self, resources):
+        client = local_session(self.manager.session_factory).client('mailmanager')
+
+        def _augment(r):
+            tags = client.list_tags_for_resource(ResourceArn=r['IngressPointArn'])['Tags']
+            r['Tags'] = tags
+            return r
+        resources = super().augment(resources)
+        return list(map(_augment, resources))
+
+
+@resources.register('ses-ingress-endpoint')
+class SESIngressEndpoint(QueryResourceManager):
+    class resource_type(TypeInfo):
+        service = 'mailmanager'
+        enum_spec = ('list_ingress_points', 'IngressPoints', None)
+        detail_spec = ('get_ingress_point', 'IngressPointId', 'IngressPointId', None)
+        name = 'IngressPointName'
+        id = 'IngressPointId'
+        arn_type = 'mailmanager-ingress-point'
+        arn = 'IngressPointArn'
+        universal_taggable = object()
```

Comment:
We should probably remove this line, since it implies we can use the Resource Groups Tagging API to manage tags. We'll be using the tag actions you've manually defined instead.

Reply:
Thanks, code updated